tests/test_cli.py

```diff
@@ -124,8 +131,41 @@ def test_file_is_ftped(
     assert os.path.exists(os.path.join(ftp_directory, "people.xml"))
 
 
-def test_cli_database_connection_success(caplog, runner):
-    result = runner.invoke(main, ["--database_connection_test"])
+def test_cli_connection_tests_success(caplog, ftp_server, monkeypatch, runner):
+    ftp_socket, _ = ftp_server
+    monkeypatch.setenv(
+        "SYMPLECTIC_FTP_JSON",
+        (
+            '{"SYMPLECTIC_FTP_HOST": "localhost", '
+            f'"SYMPLECTIC_FTP_PORT": "{ftp_socket[1]}",'
+            '"SYMPLECTIC_FTP_USER": "user", '
+            '"SYMPLECTIC_FTP_PASS": "pass"}'
+        ),
+    )
```

Comment:
This is repeated several times in `test_cli`, can we make it a fixture? You can leave the block in `test_cli_connection_tests_fail` since you have to foul up the password

Reply:
Please see the latest commit. I used the FTP server fixture when setting up the test environment (i.e.m=, `_test_env` fixture).